doc/frameworks/xgboost/using_xgboost.rst

```diff
@@ -390,6 +390,57 @@ The function should return a byte array of data serialized to ``content_type``.
 The default implementation expects ``prediction`` to be a NumPy array and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
+Bring your own model
+--------------------
+
+You can deploy an XGBoost model that you trained outside of SageMaker by using the Amazon SageMaker XGBoost container.
+Typically, you save an XGBoost model by pickling the ``Booster`` object or calling ``booster.save_model``.
+The XGBoost `built-in algorithm mode <https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html#xgboost-modes>`_
+supports both a pickled ``Booster`` object and a model produced by ``booster.save_model``.
+For a sample notebook that shows to use the XGBoost built-in algorith mode to load a pre-existing XGBoost model,
+see `Amazon SageMaker XGBoost Bring Your Own Model <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/xgboost_bring_your_own_model/xgboost_bring_your_own_model.ipynb>`_.
+
+You can also deploy an XGBoost model by using XGBoost as a framework.
+By using XGBoost as a framework, you have more flexibility.
+To deploy an XGBoost model by using XGBoost as a framework, you need to:
+- Write an inference script.
+- Create the XGBoostModel object.
+
+Write an inference script
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You must create an inference script that implements (at least) the ``model_fn`` function that calls the loaded model to get a prediction.
+
+Optionally, you can also implement ``input_fn`` and ``output_fn`` to process input and output,
+and ``predict_fn`` to customize how the model server gets predictions from the loaded model.
+For information about how to write an inference script, see `SageMaker XGBoost Model Server <#sagemaker-xgboost-model-server>`_.
+Pass the filename of the inference script as the ``entry_point`` parameter when you create the `XGBoostModel` object.
+
+Create a XGBoostModel object
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now call the ``sagemaker.xgboost.model.XGBoostModel`` constructor to create a model object,
+and then call its ``deploy()`` method to deploy your model for inference.
+
+.. code:: python
+
+    from sagemaker import get_execution_role
+    role = get_execution_role()
+
+    xgboos_model = XGBoostModel(
+        model_data="s3://my-bucket/my-path/model.tar.gz",
+        role=role,
```

Comment:
`get_execution_role` isn't really relevant to this code snippet, and since it's intended to be used only in notebooks, I think it's more distracting than helpful here.

```suggestion
    xgboost_model = XGBoostModel(
        model_data="s3://my-bucket/my-path/model.tar.gz",
        role="my-role",
```